ruby/iteration.rb

```diff
@@ -0,0 +1,83 @@
+def blockhead
+	puts "Hey blockhead!"
+	yield
+end
+
+blockhead { puts "Wassup?" }
+
+godfather = ["Vito", "Sonny", "Michael", "Fredo"]
+	puts godfather
+
+godfather.each { |character| puts "#{character} is my favorite Godfather character!" }
```

Comment:
There is nothing inherently with this line just make sure that you understand the `puts` returns `nil`